Content.Server/Access/Systems/IdCardConsoleSystem.cs

```diff
@@ -129,6 +129,14 @@ private void UpdateUserInterface(EntityUid uid, IdCardConsoleComponent component
         _idCard.TryChangeFullName(targetId, newFullName, player: player);
         _idCard.TryChangeJobTitle(targetId, newJobTitle, player: player);
 
+        JobPrototype? job = null;
+        if (!string.IsNullOrWhiteSpace(newJobProto)
```

Comment:
Not sure if I should do this sanity check

Reply:
See my comment on TryChangeJobIcon(), then you shouldn't need to do this.